rdrf/rdrf/helpers/utils.py

```diff
@@ -705,6 +705,31 @@ def get_supported_languages():
     return [Language(pair[0], pair[1]) for pair in settings.LANGUAGES]
 
 
+def get_all_language_codes():
+    from collections import namedtuple
+    from django.conf import settings
+    from langcodes import Language, standardize_tag, LANGUAGE_ALPHA3
+
+    languages = []
+    language_info = namedtuple('Language', ['code', 'name'])
+    languages_in_settings = dict(settings.LANGUAGES)
+    language_codes = list(languages_in_settings.keys())
+    extra_language_codes = LANGUAGE_ALPHA3.keys()
+    language_codes.extend(code for code in extra_language_codes if code not in language_codes)
```

Comment:
fyi: should use `set`s here to make combining the languages faster, but see comment above regarding refactoring this function.